### ISSUE TITLE
Single letter tenants can't have custom persistence objects in same declaration

### ISSUE THREAD

**Issue by codatory** [F5Networks/f5-appsvcs-extension#613]
### Environment
 * Application Services Version: 3.36.0 - confirmed on 3.36.1
 * BIG-IP Version: 14.1.4.6 & 15.1.5.1

### Summary
When declaring a tenant with a single-letter name and a custom persistence object the declaration fails with the error "serviceMain/persistenceMethods/0/use: contains an invalid path (invalid character)". This declaration passes schema validation. Adding a letter to the name or removing the custom persistence object (including using the built-in objects) works fine.

This is obviously easily worked around.

### Steps To Reproduce
1. Submit the following declaration:
```json
{
  "$schema": "https://raw.githubusercontent.com/F5Networks/f5-appsvcs-extension/master/schema/latest/as3-schema.json",
  "class": "AS3",
  "action": "deploy",
  "persist": true,
  "declaration": {
    "class": "ADC",
    "schemaVersion": "3.36.0",
    "id": "urn:uuid:33045210-3ab8-4636-9b2a-c98d22ab915d",
    "label": "F",
    "remark": "Example HTTP application with RR pool",
    "F": {
      "class": "Tenant",
      "A1": {
        "class": "Application",
        "template": "http",
        "serviceMain": {
          "class": "Service_HTTP",
          "virtualAddresses": [
            "192.0.2.99"
          ],
          "persistenceMethods": [
              {"use": "source-ip"}
          ],
          "pool": "web_pool"
        },
        "source-ip": {
            "class": "Persist",
            "persistenceMethod": "source-address"
        },
        "web_pool": {
          "class": "Pool",
          "monitors": [
            "http"
          ],
          "members": [
            {
              "servicePort": 80,
              "serverAddresses": [
                "192.168.1.10",
                "192.168.1.11"
              ]
            }
          ]
        }
      }
    }
  }
}
```

2. Observe the following error response:
```json
{
    "id": "d9491540-6829-484d-90cc-39ff83f821ae",
    "results": [
        {
            "code": 422,
            "errors": [
                "/F/A1/serviceMain/persistenceMethods/0/use: contains an invalid path (invalid character)"
            ],
            "declarationFullId": "",
            "message": "declaration is invalid"
        }
    ],
    "declaration": {}
}
```

### Expected Behavior
Either the declaration fails schema validation or applies correctly.

### Actual Behavior
Fails with a "declaration is invalid" error.

**Comment by sunitharonan** [comment]
Please contact us at  automation_toolchain_pm@f5.com so we can prioritize.

**Comment by sunitharonan** [comment]
This has been resolved in the 3.41.0 release of AS3.
